src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'cd-dashboard-help',
+  templateUrl: './dashboard-help.component.html',
+  styleUrls: ['./dashboard-help.component.scss']
+})
+export class DashboardHelpComponent implements OnInit {
```

Comment:
Did you already have prettified the code?

Reply:
@votdev Thanks for the reminder. Done.